src/components/Course.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+import { Card, Typography } from "antd";
+import "../styles/Course.css";
+
+interface CourseProps {
+  courseName: string;
+}
+
+const Dimension: React.FC<CourseProps> = (props: CourseProps) => {
+  const onClick = () => {
+    // TODO: Redirect to appropriate course page
```

Comment:
Is this going to be a different ticket (the course page on the dashboard)? It's not currently on the Jira backlog

Reply:
Yeah I think it is, I'll add a new ticket for it